src/components/OrganizationsList/index.jsx

```diff
@@ -1,159 +1,171 @@
 import React, {Component} from 'react';
-import { Table, Button, InputGroup, InputGroupAddon, Input } from 'reactstrap';
 import Api from '../../utils/api';
-import Item from './Item';
 import AlertMessage from '../../sharedComponents/AlertMessage';
-import { Pagination, PaginationItem, PaginationLink } from 'reactstrap';
+import ReactTable from 'react-table';
+import 'react-table/react-table.css';
+import './OrganizationsList.css'
 
 export default class DriversList extends Component {
 
-  state = {
-    organizations: [],
-    errors: null,
-    flash: null,
-    selectedPage: 1,
-    name: ''
+  constructor() {
+    super()
+    this.state = {
+      taxiSitesRaw: [],
+      taxiSitesList: [],
+      loading: true,
+      currentPage: 0,
+      pages: 0,
+      errors: null,
+      name: '',
+      flash: null,
+      siteName: ''
+    }
   }
 
   componentDidMount() {
-    const flash = this.props.location.flash;
-    if (flash) this.setState({flash});
-    Api.get('/organizations')
+    const flash = this.props.location.flash
+    if(flash) {
+      this.setState({flash})
+    }
+    this.fetchTaxiSites()
+  }
+
+  confirmDelete = (siteId) => {
+    var option = window.confirm("Eliminar?")
+    if (option === true) {
+      this.deleteTaxiSite(siteId)
+    }
+  }
+
+  fetchTaxiSites = () => {
+    this.setState({ loading: true })
+    Api.get(`/organizations?page=${this.state.currentPage+1}`)
       .then(res => {
-        const {organizations, pageCount} = res.data;
         this.setState({
-          organizations,
-          pageCount
+          taxiSitesRaw: res.data.organizations,
+          pages: res.data.pageCount,
+          loading: false
+        }, () => {
+          this.formatFetchedSites(this.state.taxiSitesRaw)
         })
-      });
+    }).catch((err) => {
+      this.setState({
+        errors: err.response.data.errors
+      })
+    }) 
+  }
+
+  formatFetchedSites = (taxiSites) => {
+    var data = []
```

Comment:
Yep I might went a little over the top there. Leaving it as a const implies that It will not change while the component exists, and I created the var there to make sure that the variable that will save the sites is initialized empty every time the function is called.

Reply:
if you need something that will change, use `let` 😄

Reply:
tru